modules/common/src/main/resources/application.conf

```diff
@@ -7,6 +7,7 @@
     "windowing": "10 minutes"
     "output": {
         "compression": "GZIP"
+        "wideRow": false
```

Comment:
What would it mean if we have `formats.defaul=TSV` and `json=[some-schemas]` and `wideRow=true`? To me it looks like ambigous config. In other words, I think `output.wideRow` should be mutually exclusive with everything in `formats`. Either we shred the data into some formats OR we transform he data into wide-row format.

Perhaps a union type like:

```
    "formats": {
        "type": "wideRow"
    }
```

vs

```
    "formats": {
        "type": "shred",
        "default": "TSV"
        "json": [],
        "tsv": []
    }
```

Also, don't mind it to be `output`